src/components/Rockets/Rocket.jsx

```diff
@@ -0,0 +1,48 @@
+import PropTypes from 'prop-types';
+import { useDispatch } from 'react-redux';
+import Button from 'react-bootstrap/Button';
+import Card from 'react-bootstrap/Card';
+import Badge from 'react-bootstrap/Badge';
+import { rocketBooking, rocketCancelation } from '../../redux/rockets/rockets';
```

Comment:
- [ ] Nice work done on your Rockets component, kindly check this import `../../redux/rockets/rockets, I had issues starting your project because the name of the folder starts with a capital `R`edux and you imported it using a Lowercase `r`edux